dijet/selection/default.py

```diff
@@ -21,6 +21,7 @@
 from dijet.production.dijet_balance import dijet_balance
 from dijet.production.jet_assignment import jet_assignment
 from dijet.selection.jet_selection import jet_selection
+from dijet.selection.lepton_selection import lepton_selection
```

Comment:
Maybe also a question to @dsavoiu, should we leave the "_selection" part out of the files name to keep everything short ? So something like "from dijet.selection.lepton import lepton_selection". Also for "jet_selection then".

Reply:
Yes, that's a good idea. The `selection` part should be clear from the module name, and we can just have `jet.py` and `lepton.py` as the filenames.

Reply:
Yes, that's a good idea. The selection part should be clear from the module name, and we can just have `jet.py` and `lepton.py` as the filenames.